github.go

```diff
@@ -273,6 +295,13 @@ func (m *GithubClient) GetPullRequest(prNumber, commitRef string) (*PullRequest,
 						}
 					}
 				} `graphql:"commits(last:$commitsLast)"`
+				Labels struct {
```

Comment:
Hm, are we certain we need to fetch the label information again for `get` if we are not really doing anything with it?

Reply:
Please correct me if I am wrong but I assumed we could only populate the metadata section after we do a `get`. 

However based on the comment below I decided to remove it completely from the metadata so we don't end up doing this extra label call.